spec/i18n_spec.rb

```diff
@@ -7,7 +7,7 @@
     ENV["ENFORCED_LOCALES"].presence || "en"
   end
 
-  let(:i18n) { I18n::Tasks::BaseTask.new({ locales: locales.split(",") }) }
+  let(:i18n) { I18n::Tasks::BaseTask.new }
```

Comment:
```suggestion
  let(:i18n) { I18n::Tasks::BaseTask.new(locales: locales.split(",")) }
```
Let's keep the locales, only update the way to pass the vars